kubernetes/loculus/templates/_common-metadata.tpl

```diff
@@ -26,8 +26,9 @@ fields:
     header: Submission details
     enableSubstringSearch: true
   - name: isRevocation
+    displayName: Is revocation
```

Comment:
This is a weird name to non-programmers but I think it's OK for now

Reply:
agreed but the only other boolean field I found was isLabHost (or something similar) and that was also like that.
Good of you to point out! I'll resolve for now.